deploy/vagrant/Vagrantfile

```diff
@@ -0,0 +1,10 @@
+Vagrant.configure('2') do |config|
+  config.vm.box = 'centos/6'
```

Comment:
Hmmmmm... That is for testing the RPM packaging. It wasn't meant to test counter itself. I doubt Counter is never going to run on 7. But maybe?

Why don't we worry about it if it ever happens.